api/test/regression/database_users.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 func TestDatabaseUsersModelRegression() {
-	_ = mongodbatlas.PaginatedApiAtlasDatabaseUserView{
```

Comment:
Leftover from the previous PR. Renamse for models now failed build due to models were still present.